CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 ## [Unreleased] - ReleaseDate
 
+### Added
+
+- [#62] Added an SDL based audio example (audio.rs).
```

Comment:
What do you think about renaming the example to `sdl_audio.rs` to make it clearer that this is more about SDL than a feature of the simulator itself?

Reply:
Sure! Renamed to sdl-audio.rs with `-` to be aligned with other example filenames.